initializr-service/deploy-initializr-service.cmd

```diff
@@ -0,0 +1,4 @@
+@ECHO OFF
+
+cd ../ && mvn clean package -Dmaven.test.skip=true -Pfull && cd initializr-service
+docker push ${your-acr-username}/azure-spring-cloud:latest && mvn azure-webapp:deploy
```

Comment:
I suggest to have another build label such as `build time`. Then we'll have all historic images in container registry in case we want to roll back to old version.

Reply:
make sense, I will refine this.

Reply:
good idea!